src/controllers/transfer/transfer.ts

```diff
@@ -635,7 +635,9 @@ export class TransferController extends EventEmitter {
       this.emitUpdate()
     })
     this.signAccountOpController.onError((error) => {
-      this.#portfolio.overridePendingResults(this.signAccountOpController!.accountOp)
+      if (this.signAccountOpController?.accountOp)
```

Comment:
if statement should be:
```
if (this.signAccountOpController)
```
as accountOp is always true if signAccountOpController is initialized

Reply:
Thanks, you're right, improved in https://github.com/AmbireTech/ambire-common/pull/1604/commits/b4c7a116942c3fc321f31457eee2186dbe9e8594